lib/spack/spack/modules/common.py

```diff
@@ -48,6 +48,8 @@
 import spack.error
 import spack.util.spack_yaml as syaml
 
+from spack.package_prefs import get_package_permissions, get_package_group
```

Comment:
don't import the symbols -- import the module.  Trying to keep the load times down and trying to avoid circular dependencies.

Reply:
import it `as pp` or something short if you need to.